aea/cli/add.py

```diff
@@ -123,23 +127,28 @@ def _add_item(click_context, item_type, item_public_id) -> None:
         sys.exit(1)
 
     # find and add protocol
-    if is_local:
+    if item_public_id in [DEFAULT_CONNECTION, DEFAULT_PROTOCOL, DEFAULT_SKILL]:
```

Comment:
This is the change we discussed @panasevychol, the special packages are taken from the distribution.